dotcom-rendering/src/web/lib/useAB.ts

```diff
@@ -1,9 +1,14 @@
-import type { ABTestAPI } from '@guardian/ab-core';
+import type { ABTestAPI, Participations } from '@guardian/ab-core';
 import { mutate } from 'swr';
 import useSWRImmutable from 'swr/immutable';
 
-const apiPromise = new Promise<ABTestAPI>(() => {});
-const key = 'ab-test-api';
+type ABTests = {
+	api: ABTestAPI;
+	participations: Participations;
+};
+
+const apiPromise = new Promise<ABTests>(() => {});
+const key = 'ab-tests';
```

Comment:
Wondering if we could simply add the participations as an extra key?

```ts
type ABTests = ABTestAPI & {
	participations: Participations;
};

const apiPromise = new Promise<ABTests>(() => {});
const key = 'ab-tests';
```

Reply:
I tried this but because AB is a class I am not able to spread the API functions to build another object with participations that satisfies `ABTests`.

Manually picking out the API functions doesn't feel great either so will keep as is for now... unless there is some TS wizardry that I'm missing 🤔

Reply:
Fair enough! As-is isn't bad.